### PR TITLE
Feature: Toggle (round 2)

### PR DESCRIPTION
**Pull request by CSKingMartin**
Fresh PR for Toggle work after resolving: https://github.com/CSKingMartin/project-minotaur/pull/5

### CODE REVIEW

**Comment by Brookeruu** [comment]
Resolving issues from original PR:
- Removed global link css
- Changed ‘.switch’ class to ’.Toggle’
- Refactored css code: alphabetized, combined to make more DRY, changed 'px' to 'rem'
- Removed unnecessary imported files in the mdx file
- In index.js, moved passing props from componentDidMount() to render()
- Shortened logic for toggleSwitch(),  added semicolons for linter, added () to actually call this.props.onChange()
- Created ToggleExample.jsx to provide example of Toggle state being passed into/used by another component, which is being displayed in mdx file

Testing:
please make a fresh pull
install with yarn or npm i
run yarn dev
travel to the index page and click on Toggle

toggle-gif

File: src/molecules/Toggle/Toggle.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.Toggle {
+  margin: 1rem;
+  cursor: pointer;
```
Comment (CSKingMartin) [inline src/molecules/Toggle/Toggle.css]: Can you check on how this style is being applied? I get a `pointer` cursor when hovering to the side of your Toggle, but not when hovering the UI.

Also - could you please do a quick pass through your CSS again to remove comments and alphabetize?

File: src/molecules/Toggle/ToggleExample.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (CSKingMartin) [inline src/molecules/Toggle/ToggleExample.jsx]: OCD comments:
- please remove empty new line at the top of the file
- please make sure there's one extra line at the end of the file

More reasonable comments:
- I like your intentional separation here, but it'd be better, I think, if you just defined this `ToggleExample` object from your `index.js` file

File: src/molecules/Toggle/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import ToggleExample from './ToggleExample.jsx';
```
Comment (CSKingMartin) [inline src/molecules/Toggle/index.js]: Would you rename this file to `index.jsx` for consistency?

File: src/molecules/Toggle/ToggleExample.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+const ToggleExample = (props) => {
+  const {
+    className,
+    children,
+    ...rest
+  } = props;
+
+  return(
+    <div>
+      {props.toggle ?
```
Comment (CSKingMartin) [inline src/molecules/Toggle/ToggleExample.jsx]: You have `props.toggle` here, but haven't defined them above in `props`

File: src/molecules/Toggle/ToggleExample.jsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  return(
+    <div>
+      {props.toggle ? 
+        <p className="Toggle__on">Toggle switch "ON"</p> :
+        <p className="Toggle__off">Toggle switch "OFF"</p>
+      }     
+      </div>
+  )
+
```
Comment (CSKingMartin) [inline src/molecules/Toggle/ToggleExample.jsx]: Can you add some PropTypes validation here?

**Review by CSKingMartin — COMMENTED** [review]
Hey @Brookeruu - could you please make the described changes?
Please ping me when this is finished, and I think we can go ahead and merge this in

File: src/molecules/Toggle/ToggleExample.jsx (hunk 5 above)
Comment (CSKingMartin) [reply src/molecules/Toggle/ToggleExample.jsx]: @Brookeruu - could you define the PropTypes for the `onChange` prop as well? After you have done so, please feel free to merge / delete this branch.

If you want me to show you had to do that, please ping me and I'll show you :)

You also have some merge conflicts that need to be resolved, let me know if I can help with that either.

**Pull request merged**